scripts/download_gherkin_features.bat

```diff
@@ -0,0 +1,13 @@
+: Adding gherkin feature download for windows
+@echo off
+SET TMP_FOLDER=.\tests\tempFeatures
+SET "FOLDER=%1"
+IF "%FOLDER%"=="" SET /P "FOLDER=.\tests\bdd\features"
```

Comment:
@v1v This file is an exact copy of the original one, but supporting overriding the FOLDER variable with the 1st argument